### DISCUSSION TITLE
Feature Request: Software Install Component using winget or choco

### DISCUSSION THREAD

**Discussion by Nerdy-Technician** [immense/Remotely#923] [General]
**Is your feature request related to a problem? Please describe.**
Feature request

**Describe the solution you'd like**
An option to deploy software to a machine with winget without having to use script but have user freindly names for the apps 

**Describe alternatives you've considered**
Winget scripts however its not sustainable for my agents

**Comment by dkattan** [comment]
Hey @Brandon-Roff checkout https://immy.bot/ it has Remotely’s Remote Control engine built in and is designed for software installation and configuration.

**Comment by Nerdy-Technician** [comment]
Good Afternoon,

no i mean for remotely I can warrant $400 I would like to see it in remotely

**Comment by Nerdy-Technician** [comment]
Please dont close the case until i am happy please

**Comment by Nerdy-Technician** [comment]
Can we please reopen this case

**Comment by dkattan** [comment]
I'm happy to discuss the merits of this feature here. I understand you are willing to spend $400/mo for this solution. Have you done a trial on our ImmyBot product? It is the primary focus of our development.

**Reply by Nerdy-Technician** [reply]
I use remotely for personal devices and manage other devices if im gonna spend that much ill use datto rmm

**Reply by dkattan** [reply]
The problem with DattoRMM is it takes ~5 minutes to execute anything on devices. ImmyBot, like Remotely, maintains a websocket to all the machines and is able to run scripts and software immediately. I know you probably got the wrong vibe with me closing your issue and everything, but I promise if you check it out you'll look at Datto RMM like yesterday's garbage, because it is really CentraStage a nearly 20 year old product with a UI facelift, developed before Websockets or any modern "at-scale" tech.

**Reply by Nerdy-Technician** [reply]
I am happy with the 5 minute delay it means if someone runs a task at fault it can be blocked, I apperaite it however I am more intrested in the remotely aspect to have it to install software for family members more than a dedicated RMM,Centrastage may be old and indeed has its issues but it works well components and variables very easy to solve problems
